jobs/updater/dev-public-catalog/values.yaml

```
@@ -0,0 +1,27 @@
+name: "updater"
+
+serviceAccount:
+  roleArn: "arn:aws:iam::505630707203:role/interop-public-catalog-updater-dev"
```

```suggestion
  roleArn: "arn:aws:iam::{{.Values.local.awsAccountId}}:role/interop-public-catalog-updater-{{.Values.local.awsEnvironmentName}}"
```
create locals in values-microservice, values-cronjob